subsys/bluetooth/host/hci_core.c

```diff
@@ -3086,6 +3086,15 @@ static void le_read_supp_states_complete(struct net_buf *buf)
 	bt_dev.le.states = sys_get_le64(rp->le_states);
 }
 
+static void le_read_maximum_adv_data_len_complete(struct net_buf *buf)
+{
+	struct bt_hci_rp_le_read_max_adv_data_len *rp = (void *)buf->data;
+
+	LOG_DBG("status 0x%02x", rp->status);
+
+	bt_dev.le.max_adv_data_len = rp->max_adv_data_len;
```

Comment:
Don't you have to do byte order conversion here? I.e. looks like a missing `sys_le16_to_cpu()`.

Reply:
Thanks, I forgot it